### PR TITLE
Fixed bug where this basically didn't work at all.

### PR DESCRIPTION
**Pull request by arb**
Yeah this wasn't working at ALL in version 3. When this is merged, I'm going to unpublished version 3.

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ var internals = {
 internals.store.lookUp = function (obj, method) {
   var i = this.length;
   var lookup;
-  while (i-- && !lookup) {
+  while (!lookup && i--) {
```
Comment (cjihrig) [inline lib/index.js:13]: Should have just used a `break` statement :trollface:

Comment (arb) [inline lib/index.js:13]: `break` is a hack. I know it, you know it, the cat knows it.

Comment (cjihrig) [inline lib/index.js:13]: I guess we have to agree to disagree. I can't take that statement seriously given it's use in everything from Node core to hapi :-)

Comment (arb) [inline lib/index.js:13]: The only reason people use `break` is because they have the `for` loop on speed dial and don't think of using a `while` loop.

Comment (cjihrig) [inline lib/index.js:13]: It's more readable than maintaining another variable and having a more complex check on every iteration of the loop.

**Pull request merged**